src/mol-math/geometry/lookup3d/grid.ts

```diff
@@ -40,6 +40,40 @@ class GridLookup3DImpl<T extends number = number> implements GridLookup3D<T> {
         return ret;
     }
 
+    nearest(x: number, y: number, z: number): { index: number, squaredDistance: number } | undefined {
```

Comment:
Maybe this function should be ``nearest(x: number, y: number, z: number, k: number): Result<T>`` instead and return ``k``-nearest points. This will make it possible to find a nearest atom to another one within a structure (by setting ``k=2`` and taking the further one).